### ISSUE TITLE
valgrind on Monterey version 12.2.1 with Apple M1 Max chip

### ISSUE THREAD

**Issue by GeraldCNelson** [LouisBrunner/homebrew-valgrind#9]
I attempted to install on my new mac but was told the kernel version is unsupported. I'm checking to see if there is any plan to support it.

**Comment by LanYunDev** [comment]
I also encountered the same problem, I hope to solve soon!

**Comment by ghost** [comment]
I'm on a x86 MacBook Pro running Monterey, and I encounter issues when downloading as well. Not sure if I'm just having an isolated issue, or if tool is not supported on MacOS Monterey.

**Comment by wvdgoot** [comment]
Both MacOS 11+ and the M1 chipset are not supported: see issue https://github.com/LouisBrunner/homebrew-valgrind/issues/8 and https://github.com/LouisBrunner/homebrew-valgrind/issues/5.

**Comment by GeraldCNelson** [comment]
What I ended up doing was moving the code to a linux box and running it there. No particular technical challenges (just have to have a linux system). Then I ran the following code (or variants). 
`R -d "valgrind -s" code/test_for_valgrind.R`
To start off cd to the project directory. 
R - run R at the command line - R needs to be installed
d - run R in debug code "valgrind -s" - run valgrind with the s option. valgrind needs to be installed.
code/test_for_valgrind.R - run a simple R script in the code directory.

Hope this process description is useful to others.
